econml/dml.py

```diff
@@ -393,48 +474,39 @@ class SparseLinearDMLCateEstimator(_DMLCateEstimatorBase):
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
-
-    inference: string, inference method, or None
-        Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`)
     """
 
     def __init__(self,
                  linear_model_y=LassoCV(), linear_model_t=LassoCV(), model_final=LinearRegression(fit_intercept=False),
```

Comment:
I think we should not change the input parameter names for consistency. So we should keep using model_y and model_t. Especially because we are not saying that the first stages have to be sparse linear. The could also be forests. The special thing here is that the final stage should be a lassoCV. This should also change. Here we should have LassoCV as the final model. Not Linear Regression. Otherwise it's not really sparse linear.

Reply:
Sure, that's reasonable.

Reply:
@vasilismsr There's an anooying complication here: `LassoCV` doesn't support multiple outputs, while `MultiTaskLassoCV` *only* supports multiple outputs, so there isn't a single model we can use for this class unless we write our own that delegates to the appropriate one based on the dimensionality of `Y`.

Reply:
I would go with the 'auto' strategy which selects the model at fit time based on the dimensionality. Similarly for the propensity the 'auto' could select between lassoCV or logisticRegressionCV, based on whether treatment is discrete or not. 'auto' is somewhat standard for defaults, in my opinion (e.g. max_features of https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html )

Reply:
your wrapper approach also looks good.